core/citrus-api/src/test/java/org/citrusframework/spi/ClassPathResourceResolverTest.java

```diff
@@ -73,9 +100,17 @@ public Enumeration<URL> getResources(String name) throws IOException {
 
             if (name.equals("META-INF/citrus/test/parser/core/")) {
                 URL url = delegate.getResource(baseJar);
+                Objects.requireNonNull(url);
```

Comment:
```suggestion
                requireNonNull(url);
```

twice, also https://github.com/citrusframework/citrus/pull/1096/files#diff-6668f866353876072e237647ff170d14eb06f78ff5e5eaea7ddbf98e84f9ba6cR135.